src/sentiment.py

```diff
@@ -0,0 +1,302 @@
+# Code adapted from https://github.com/Himabindugssn/Sentiment-classification-using-transformers
+
+from tqdm import tqdm
+import pandas as pd
+import numpy as np
+from sklearn.model_selection import train_test_split
+from sklearn.utils.class_weight import compute_class_weight
+from sklearn.metrics import classification_report
+import torch
+import torch.nn as nn
+from torch.utils.data import TensorDataset, DataLoader, RandomSampler, SequentialSampler
+from transformers import AdamW, AutoModel, BertTokenizerFast
+
+device = torch.device("cuda")
+
+
+LR = 1e-5  # Learning rate
+
+
+class BERT_architecture(nn.Module):
+    def __init__(self, bert):
+        super(BERT_architecture, self).__init__()
```

Comment:
This architecture is more complicated than I wrote in the report! Please make sure the report mentions that your architecture uses 2 feed forward network with dropout and relu between the.